### ISSUE TITLE
Buttons Must have discernible text for screen readers

### ISSUE THREAD

**Issue by MundiaNderi** [ushahidi/platform#4761]
### \### Element does not have inner text that is visible to screen readers.

![Screenshot 2024-03-07 at 11 58 20](https://uploads.linear.app/f1df6e34-3bca-4ade-bb48-e6a457e64091/f07e8e9b-37c5-43fb-bf9e-13b5f5740ec2/a9036945-f1e0-4d28-8cc0-ebf7dcc11802?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiL2YxZGY2ZTM0LTNiY2EtNGFkZS1iYjQ4LWU2YTQ1N2U2NDA5MS9mMDdlOGU5Yi0zN2M1LTQzZmItYmY5ZS0xM2I1ZjU3NDBlYzIvYTkwMzY5NDUtZjFlMC00ZDI4LThjYzAtZWJmN2RjYzExODAyIiwiaWF0IjoxNzE3MTkyNDY1LCJleHAiOjMzMjg3NzUyNDY1fQ.2CuV3rQKwzHJa-9cwTXeiEPwpK_SvlKYC5aPvprlDdo "=647x")

**Comment by MundiaNderi** [comment]
@Angamanga I have created this issue based on the instructions here but I'm stuck on how to label the  issue :https://github.com/ushahidi/platform/issues/4750
